mage/main.go

```diff
@@ -27,6 +27,9 @@ const mageVer = "v0.3"
 
 var output = template.Must(template.New("").Funcs(map[string]interface{}{
 	"lower": strings.ToLower,
+	"lowerfirst": func(s string) string {
+		return strings.ToLower(string(s[0])) + s[1:]
```

Comment:
this is probably fine 99% of the time, but it technically can break with unicode characters that are more than a single byte, since slicing a string returns bytes, not unicode codepoints (runes).  

A safer (if less elegant) way to do it is 

```go
r := []rune(s)
r = append([]rune{unicode.ToLower(r[0])}, r[1:])
return string(r)
```

Reply:
I did stumble across a rune example doing this, though I failed to spot that the slice would break with unicode chars. Thanks for noting.